src/components/WorkspaceAdd/WorkspaceAddButton.tsx

```diff
@@ -117,7 +117,7 @@ const WorkspaceAddButton = ({ workspaces }: { workspaces: K8sResourceCommon[] })
           </Button>,
         ]}
       >
-        {error ? <Alert variant="danger" isInline title={error} role="alert" /> : null}
+        {error ? <Alert variant="danger" isInline title={error} role="alert" titleHeadingLevel="h2" /> : null}
```

Comment:
This fixes an issue that was found when the evergreen tests for WorkspaceAddButton were fixed